exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/metrics_exporter/__init__.py

```diff
@@ -54,48 +54,54 @@
     MetricsExporter,
     MetricsExportResult,
 )
+from opentelemetry.sdk.metrics.export.aggregate import (
+    HistogramAggregator,
+    LastValueAggregator,
+    MinMaxSumCountAggregator,
+    SumAggregator,
+    ValueObserverAggregator,
+)
 
 logger = logging.getLogger(__name__)
 DataPointT = TypeVar("DataPointT", IntDataPoint, DoubleDataPoint)
 
 
 def _get_data_points(
-    sdk_metric: MetricRecord, data_point_class: Type[DataPointT]
+    sdk_metric_record: MetricRecord, data_point_class: Type[DataPointT]
 ) -> List[DataPointT]:
 
-    data_points = []
-
-    for (
-        label,
-        bound_counter,
-    ) in sdk_metric.instrument.bound_instruments.items():
-
-        string_key_values = []
-
-        for label_key, label_value in label:
-            string_key_values.append(
-                StringKeyValue(key=label_key, value=label_value)
-            )
-
-        for view_data in bound_counter.view_datas:
-
-            if view_data.labels == label:
-
-                data_points.append(
-                    data_point_class(
-                        labels=string_key_values,
-                        value=view_data.aggregator.current,
-                        start_time_unix_nano=(
-                            view_data.aggregator.last_checkpoint_timestamp
-                        ),
-                        time_unix_nano=(
-                            view_data.aggregator.last_update_timestamp
-                        ),
-                    )
-                )
-                break
-
-    return data_points
+    if isinstance(sdk_metric_record.aggregator, SumAggregator):
+        value = sdk_metric_record.aggregator.checkpoint
+
+    elif isinstance(sdk_metric_record.aggregator, MinMaxSumCountAggregator):
+        # FIXME: How are values to be interpreted from this aggregator?
```

Comment:
might be worth taking a look at the go implementation for this, though it doesnt have to be part of this PR if you want to open a separate issue https://github.com/open-telemetry/opentelemetry-go/blob/master/exporters/otlp/internal/transform/metric.go#L406-L540

Reply:
I think that would be better implemented in another PR along with the rest of changes that need to be implemented to enhance this PR to the Go implementation level.